tests/dualtor/test_tor_ecn.py

```diff
@@ -92,9 +95,10 @@ def build_encapsulated_ip_packet(
                         if is_ipv4_address(addr.split("/")[0])][0]
     tor_ipv4_address = tor_ipv4_address.split("/")[0]
 
-    inner_dscp = random.choice(range(0, 33))
+    inner_dscp = random.choice([3, 4, 2, 6]) #lossy queue is 3 or 4 or 2 or 6.
```

Comment:
The comment should be #lossless queue is 3 or 4 or 2 or 6.
More over, the lossless queues are 2,3 4,6 only if PCBB is enable. Otherwise, we still have 2 lossless queues (3 and 4). So we need a check here

Reply:
Even PCBB is disable, it will check Azure table, will go to correct queue, test case will not fail.
And also test 3,4,2,6 one by one, not randomly choose one.